templates/repo/create.tmpl

```diff
@@ -72,9 +72,17 @@
 								<input class="hidden" name="git_content" type="checkbox" tabindex="0" {{if .git_content}}checked{{end}}>
 								<label>{{.i18n.Tr "repo.template.git_content"}}</label>
 							</div>
+							<div class="ui checkbox">
+								<input class="hidden" name="git_hooks" type="checkbox" tabindex="0" {{if .git_hooks}}checked{{end}}>
+								<label>{{.i18n.Tr "repo.template.git_hooks"}}</label>
+							</div>
 						</div>
 						<div class="inline field">
 							<label></label>
+							<div class="ui checkbox">
```

Comment:
The hierarchy somehow looks odd. There's a `<label></label>` for webhooks but not for topics, and those two are in a different div than git_hooks.

Reply:
This is how the migration items layout is achieved as well. The empty label (as far as I can tell) simply pushes the checkboxes into the correct position.  
  
Topics have a label `<label>{{.i18n.Tr "repo.template.topics"}}</label>`